### ISSUE TITLE
Multipart's data should be written into TileEntity

### ISSUE THREAD

**Issue by Licshee** [Chicken-Bones/ForgeMultipart#145]
Instead of writing directly into chunks' NBT data, writing into TileEntity makes it possible to do copy/paste in WorldEdit, like how it does to containers.

A conversion process may be necessary to upgrade the data from current design.

**Comment by NanaLich** [comment]
... so... is this fixed or not?

**Comment by Chicken-Bones** [comment]
I don't believe it is. I'm wary of compatibility issues with the suggested method. In 1.8 I'll just be adding the hook that was rejected from forge
